tracer/test/test-applications/integrations/Samples.AWS.DynamoDBv2/AsyncHelpers.cs

```diff
@@ -0,0 +1,257 @@
+        }
+
+        public static async Task PutItemAsync(AmazonDynamoDBClient dynamoDBClient)
+        {
+            var item = new Dictionary<string, AttributeValue>
+            {
+                ["id"] = new() { S = Guid.NewGuid().ToString() },
+                ["name"] = new() { S = "Jordan" },
+                ["lastname"] = new() { S = "González Bustamante" },
+                ["age"] = new() { N = "24" }
+            };
+
+            var putItemRequest = new PutItemRequest
+            {
+                TableName = TableName,
+                Item = item,
+            };
+
+            var response = await dynamoDBClient.PutItemAsync(putItemRequest);
+            Console.WriteLine($"PutItemAsync(PutItemRequest) HTTP status code: {response.HttpStatusCode}");
+        }
+
+        public static async Task PutItemsAsync(AmazonDynamoDBClient dynamoDBClient)
+        {
+            var person = new Dictionary<string, AttributeValue>
+            {
+                ["id"] = new() { S = "1" },
+                ["name"] = new() { S = "Jordan" },
+                ["lastname"] = new() { S = "González Bustamante" },
+                ["city"] = new() { S = "NYC" }
+            };
+            var pokemon = new Dictionary<string, AttributeValue>
+            {
+                ["id"] = new() { S = "2" },
+                ["pid"] = new() { N = "393" },
+                ["name"] = new() { S = "Piplup" },
+                ["type"] = new() { S = "water" }
+            };
+            var writeRequestList = new List<WriteRequest>
+            {
+                new() { PutRequest = new() { Item = person }, }, 
+                new() { PutRequest = new() { Item = pokemon } }
+            };
+            var batchWriteItemRequest = new BatchWriteItemRequest
+            {
+                RequestItems =
+                {
+                    { TableName, writeRequestList }
+                }
+            };
+
+            var response = await dynamoDBClient.BatchWriteItemAsync(batchWriteItemRequest);
+            Console.WriteLine($"BatchWriteItemAsync(BatchWriteItemRequest) HTTP status code: {response.HttpStatusCode}");
+        }
+
+        public static async Task GetItemAsync(AmazonDynamoDBClient dynamoDBClient)
+        {
+            var key = new Dictionary<string, AttributeValue>
+            {
+                ["id"] = new() { S = "1" },
+                ["name"] = new() { S = "Jordan" }
+            };
+            
+            var getItemRequest = new GetItemRequest
+            {
+                TableName = TableName,
+                Key = key
+            };
+            
+            var response = await dynamoDBClient.GetItemAsync(getItemRequest);
+            Console.WriteLine($"GetItemAsync(GetItemRequest) HTTP status code: {response.HttpStatusCode}");
+        }
+        
+        public static async Task GetItemsAsync(AmazonDynamoDBClient dynamoDBClient)
```

Comment:
nit: possible rename?

```suggestion
        public static async Task BatchGetItemAsync(AmazonDynamoDBClient dynamoDBClient)
```

Reply:
> I added these changes, left DeleteItemsAsync as is, since that one is a BatchWriteItem too.